src/Files.App/Helpers/Win32/Win32PInvoke.Methods.cs

```diff
@@ -115,6 +115,11 @@ public static extern bool SHGetPathFromIDList(
 			[MarshalAs(UnmanagedType.LPWStr)] StringBuilder pszPath
 		);
 
+		[DllImport("shell32.dll", CharSet = CharSet.Auto)]
```

Comment:
Could you use CsWin32?

Reply:
1. Add the method name ShellExecuteEx to Files.App.CsWin32/NativeMethods.txt
2. Add 'Windows.Win32' using directive
3. Use it as PInvoke.ShellExecuteEx();

Reply:
What about `SHELLEXECUTEINFO`?

Reply:
It's automatically added.
If the definition of it is correctly stored.

```cs
bool ShellExecuteEx(ref SHELLEXECUTEINFOW pExecInfo);
```

Reply:
I pushed a commit with these changes